From d934cdee4f92dc163fa620024b4e3a6dbbc25f37 Mon Sep 17 00:00:00 2001
From: Rui Matos <tiagomatos@gmail.com>
Date: Thu, 23 Jun 2016 15:38:03 +0200
Subject: [PATCH 6/7] ui/frames: Drop the current grab info on button release

This was added in commit d05b750b8dec3825d286e3ae30c41b41bd6aed98 and
later removed inadvertently in commit
d561b3b18f73a66bcd7947dce7ab1aab9e09965a .

https://bugzilla.gnome.org/show_bug.cgi?id=767969
---
 src/ui/frames.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ui/frames.c b/src/ui/frames.c
index 0e90567..0cb9e14 100644
--- a/src/ui/frames.c
+++ b/src/ui/frames.c
@@ -1114,6 +1114,7 @@ handle_button_release_event (MetaUIFrame *frame,
 {
   Display *display = GDK_DISPLAY_XDISPLAY (gdk_display_get_default ());
 
+  frame->frames->current_grab_op = META_GRAB_OP_NONE;
   meta_core_end_grab_op (display, event->time);
 
   /* We only handle the releases we handled the presses for (things
-- 
2.8.1

